kudu/Dockerfile

```diff
@@ -356,16 +380,16 @@ RUN curl -fL -o /usr/local/go/bin/dep $DEP_DOWNLOAD_URL \
 COPY kudu.conf /tmp
 COPY apache2.conf /tmp
 RUN npm install -g kudusync \
-  && wget -O /tmp/Kudu.zip https://nildev.blob.core.windows.net/kudu/Kudu.71.10312.3298.zip \
+  && wget -O /tmp/Kudu.zip https://patlekudu.blob.core.windows.net/kudu/Kudu.75.10626.3456.zip \
```

Comment:
Please use this URL https://wawsinfraprodbay063.blob.core.windows.net/kudu/Kudu.75.10626.3456.zip